spec/models/availability_spec.rb

```diff
@@ -34,4 +34,17 @@
       expect(availability.partially_available?(event)).to eq(false)
     end
   end
+
+  context 'cancel_duplicates' do
+    it 'sets status for previous availabilities with matching start and end times to Cancelled' do
+      start_time = Time.now
```

Comment:
We should use `Time.current` instead: https://robots.thoughtbot.com/its-about-time-zones